Set api.Element.scroll_event value for Safari

joneskj55 opened the pull request:
This PR addresses issue #9474 (scroll_event)

Fixes #9474

ddbeck reviewed (approved):
Thank you, @joneskj55! 🎉

(I rewrote the title of this PR to be a little more descriptive and added the "fixes #XYZ" text in the PR description. Now, GitHub will automatically close the issue on merge, nice and quick. ⚡)

joneskj55 commented:
> Thank you, @joneskj55! 🎉
> 
> (I rewrote the title of this PR to be a little more descriptive and added the "fixes #XYZ" text in the PR description. Now, GitHub will automatically close the issue on merge, nice and quick. ⚡)

Oh, that's cool! Will have to remember that. Thanks!
